contracts/eigenlayer/src/WavsAVSRegistrar.sol

```diff
@@ -2,30 +2,50 @@
 pragma solidity ^0.8.9;
 
 import {IAVSRegistrar} from "@eigenlayer/contracts/interfaces/IAVSRegistrar.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+// Minimal AVS Registrar contract. 
+// It allows the owner to pause the registration, preventing all Operator register and deregister operations.
+contract WavsAVSRegistrar is IAVSRegistrar, Ownable {
```

Comment:
I know this will sound pretty pedantic, but I would love to see some test code for these changes. In general, I am a big stickler for unit tests, not just to see that your code works (it seems simple enough to eyeball it), but also that no one else breaks the feature in the future.

Because this code is so trivial, I can let it slip for this PR, but please include concrete unit tests in all future PRs. The other tasks are a lot more complex and I would like solid unit test coverage for them.